timesketch/frontend-ng/src/components/Visualization/ChartCard.vue

```diff
@@ -0,0 +1,186 @@
+<!--
+ Copyright 2024 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ -->
+ <template>
+  <v-card
+    flat
```

Comment:
I would recommend to go with `outlined` instead of flat. This would add a border around the visualization that helps understand what is part of the viz and what not. (e.g. tables look a bit lost without)

Reply:
Done.  Although this results in a "double" border when creating a new visualization.

Reply:
I see. 
I think I have an idea on how to fix this and also fix the [overlapping border issue](https://github.com/google/timesketch/pull/3040#discussion_r1608588834) in one go. What do you think about the following:
1. Change the ChartCard.vue v-card back to flat.
2. Change the SavedVisualization.vue `<div>` back to a `<v-card class="mx-3" outlined>` with outlined border.
3. Change the VisualizationEditor.vue right side (TsChartCard col section) to `<v-card outlined v-show="chartSeries && selectedChartType">` so we don't have the empty v-card visible if no aggregation is created yet.

This should result in the border only showing when needed with sufficient spacing and no overlapping elements form the aggregation. 

Let me know what you think.

Reply:
Done, thanks, works for me.